rust-toolchain.toml

```diff
@@ -1,4 +1,4 @@
 [toolchain]
-channel = "stable"
+channel = "1.89.0"
```

Comment:
So, Renovate will open a PR when `stable` points to a newer version in the future?